src/workbench/InfoDialog.tsx

```diff
@@ -4,33 +4,64 @@
  * SPDX-License-Identifier: MIT
  */
 import { Button } from "@chakra-ui/button";
+import { HStack, Link, Stack, Text, VStack } from "@chakra-ui/layout";
 import {
-  Flex,
-  Link,
-  List,
-  ListItem,
-  Stack,
-  Text,
-  VStack,
-} from "@chakra-ui/layout";
-import { Modal, ModalBody, ModalContent, ModalOverlay } from "@chakra-ui/modal";
-import { RiExternalLinkLine, RiFeedbackLine } from "react-icons/ri";
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+} from "@chakra-ui/modal";
+import { Icon } from "@chakra-ui/react";
+import { ReactNode, useEffect, useState } from "react";
+import { RiExternalLinkLine } from "react-icons/ri";
+import { FormattedMessage } from "react-intl";
 import ModalCloseButton from "../common/ModalCloseButton";
+import { queryUrl } from "../common/sanity";
+import YoutubeVideoEmbed, { YoutubeVideo } from "../common/YouTubeVideo";
 import { useDeployment } from "../deployment";
+import { useLogging } from "../logging/logging-hooks";
 
 interface InfoDialogProps {
   isOpen: boolean;
   info?: boolean;
   onClose: () => void;
-  switchToInfoDialog: () => void;
 }
 
-const InfoDialog = ({
-  isOpen,
-  onClose,
-  switchToInfoDialog,
-}: InfoDialogProps) => {
+const InfoDialog = ({ isOpen, onClose }: InfoDialogProps) => {
   const { guideLink } = useDeployment();
+  const [welcomeVideo, setWelcomeVideo] = useState<YoutubeVideo | undefined>();
+  const [loadError, setLoadError] = useState<boolean>(false);
+  const logging = useLogging();
+  useEffect(() => {
+    const controller = new AbortController();
+    const { signal } = controller;
+    const fetchWelcomeVideo = async () => {
+      const query = `*[_type == "pythonEditorConfig"]{
```

Comment:
Maybe query by _id? Then it's clearer that we only expect one. _id is also "pythonEditorConfig".

Reply:
Makes sense. [`0d14718`](https://github.com/microbit-foundation/python-editor-next/pull/794/commits/0d14718730921fd631e434da706b8d7c5a5c10d9).